packages/components/src/components/colors.ts

```diff
@@ -56,7 +56,10 @@ export const colors = {
   placeholderCopy: '#959595', // grey400
   disabled: '#CCCCCC', // grey300
   background: '#F5F5F5', // grey100
-  backgroundPrimary: '#36304E'
+  backgroundPrimary: '#36304E',
+
+  // Storybook branding
+  brand: '#0058E0' // blue
```

Comment:
TODO: jpye-finch and I will try to keep the colouring consistent and will meet today about the colour system